src/prommis/leaching/tests/test_leach_flowsheet.py

```diff
@@ -17,6 +22,10 @@ def model():
     m = build_model()
     set_inputs(m)
 
+    assert number_variables(m.fs.leach) == 197
```

Comment:
These should not be part of the fixture - it looks like this might have been carried over from before I updated the leaching code.

Reply:
Yes, I believe this was carried over from before your changes. I'll remove it